iree/samples/vision/CMakeLists.txt

```diff
@@ -44,3 +44,15 @@ iree_cc_binary(
     iree::runtime
     iree::tools::utils::image_util
 )
+
+iree_lit_test_suite(
```

Comment:
Since you're just running the one test, perhaps use iree_lit_test. Unless there's a plan to add more. It will make the test names simpler

Reply:
Done.